### DISCUSSION TITLE
How to configure horizontal roof sun shade

### DISCUSSION THREAD

**Discussion by christianvaes** [basbruss/adaptive-cover#207] [Q&A] [answered]
I would like to have some help configuring my sun shade. 

I have a Velux cvu 150100 sun shade in my roof window. This looks like https://dakramen.veluxshop.nl/velux-daklichten/daklichten-voor-platte-daken/velux-cvu-0225qa it's mainly a horizontal window on the roof and the sun shade is in the roof. I can control it form home assistant. But i'm struggling how to configure this using the adaptive cover integration. Mainly because it's a bit different from the 3 options mentioned. Can you please help me how to set this up?

The main issues are:

- Should this be considered as vertical or horizontal sun shade?
- What is the azimuth? Because it's on a flat roof and sun can come in from any side.
- What to configure for the sizes(it's 150cm by 100cm)? As it's a horizontal screen on the roof so you don't really have a height.

Hopefully you can help me out.

**Comment by basbruss** [comment, marked as answer]
These covers are not covered in the integration. It really is focusing on facade windows for now. Roof windows under an angle or flat do need more geometry to determine the position. Also the workspace area can change based on the azimuth of the sun due to the rotation direction of the sunrays.

**Comment by christianvaes** [comment]
That's a pity but thank you for answering my question. Hopefully it will be supported in the future.

**Reply by KaraFR** [reply]
Same need here.  My roof windows don't react as well as my vertical windows. 

Is case being considered for futur versions ?
